### ISSUE TITLE
How to get test data

### ISSUE THREAD

**Issue by MuggleWang** [zllrunning/face-parsing.PyTorch#9]
Thanks for your wonderful work!
I have one little question. How did you get the test data. Did you split the origin dataset or something else?  Can you go over the steps in detail?
Thanks a lot!

**Comment by HulkMaker** [comment]
I have the same question, where is the test data, which named 'test-img' in your code?

**Comment by salmanhiro** [comment]
I think you can separate the training and test set by this similar training script
https://github.com/switchablenorms/CelebAMask-HQ/tree/master/face_parsing
